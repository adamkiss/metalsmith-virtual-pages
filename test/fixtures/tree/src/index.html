---
title: Non generated file
---
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>{{ title }}</title>
</head>
<body>
  This is the index of the tree fixture.
</body>
</html>