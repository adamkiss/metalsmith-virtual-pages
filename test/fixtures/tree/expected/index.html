<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Non generated file</title>
</head>
<body>
  This is the index of the tree fixture.
</body>
</html>